docs/docs/experimental/subpackages.md

```diff
@@ -381,9 +354,29 @@ podman exec -it my-audio-app ls -la /dev/snd/
 # Test GPU access with wayland-client-devices (if applicable)
 podman exec -it gpu-app ls -la /dev/dri/
 
-# Check systemd-logind seat devices (for wayland-session-devices)
-loginctl list-seats
-loginctl seat-status seat0
+# Run hook tests (if source available)
```

Comment:
Please add another section for oci-hooks docs spec files 
https://github.com/opencontainers/runtime-spec/blob/main/config.md#posix-platform-hooks

https://github.com/opencontainers/runtime-spec/blob/main/schema/config-schema.json